doc/distrib/NodeHelpFiles/_review/Revit.Elements.Room.CoreCenterBoundary.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+`Room.CoreCenterBoundary` returns a nested list representing the given room's core center boundary. In the returned list, the first sublist represents the outermost curves, while subsequent lists represent loops within the room. Core center boundaries occur on Revit rooms on the wall's section that is defined as core.
```

Comment:
Similar to previous comment. Suggest rephrasing the last line "Core center coundaries occur..." to " Core center boundaries are located at the center of the walls that are defines as core". Also a link to the room boundaries documentation would be handy : https://help.autodesk.com/view/RVT/2024/ENU/?guid=GUID-0BB62832-36DD-4E06-A9D4-EE98CE0FCF89